search-service/src/main/kotlin/db/migration/V0_33__Expand_attribute_instance_payload.kt

```diff
@@ -0,0 +1,69 @@
+// ktlint-disable filename
+
+package db.migration
+
+import com.egm.stellio.shared.util.JsonLdUtils.NGSILD_CORE_CONTEXT
+import com.egm.stellio.shared.util.JsonLdUtils.expandJsonLdFragment
+import com.egm.stellio.shared.util.JsonUtils.serializeObject
+import com.egm.stellio.shared.util.toUri
+import org.flywaydb.core.api.migration.BaseJavaMigration
+import org.flywaydb.core.api.migration.Context
+import org.slf4j.LoggerFactory
+import org.springframework.jdbc.core.JdbcTemplate
+import org.springframework.jdbc.datasource.SingleConnectionDataSource
+import java.net.URI
+
+const val LIMIT = 1000
+const val OFFSET = 0
+
+@Suppress("unused")
+class V0_33__Expand_attribute_instance_payload : BaseJavaMigration() {
+
+    private val logger = LoggerFactory.getLogger(javaClass)
+    private lateinit var jdbcTemplate: JdbcTemplate
+
+    override fun migrate(context: Context) {
+        jdbcTemplate = JdbcTemplate(SingleConnectionDataSource(context.connection, true))
+
+        queryAttributeUpdateByPage("attribute_instance", LIMIT, OFFSET)
+        queryAttributeUpdateByPage("attribute_instance_audit", LIMIT, OFFSET)
+    }
+
+    private fun queryAttributeUpdateByPage(tableName: String, limit: Int, offset: Int) {
```

Comment:
when does it stop?

Reply:
if `resultSet` retunrs no data, nothing happens then so it stops at this moment. no?

Reply:
I just stopped the build in Jenkins that was running for 4 days so no